lib/components/PrimaryButton.jsx

```diff
@@ -73,7 +73,6 @@ PrimaryButton.STATUS_HOVER = 'hover';
 
 PrimaryButton.propTypes = {
   disabled: React.PropTypes.bool,
-  onClick: React.PropTypes.function,
```

Comment:
@n8mellis this was giving an error in console:
`Failed prop type: PrimaryButton: prop type `onClick` is invalid; it must be a function, usually from React.PropTypes.
    in PrimaryButton (created by SignInForm)`